integrations/spark/build.gradle

```diff
@@ -48,6 +48,10 @@ dependencies {
     testImplementation platform('org.junit:junit-bom:5.7.1')
     testImplementation "org.postgresql:postgresql:${postgresqlVersion}"
     testImplementation 'org.hamcrest:hamcrest-library:2.2'
+    testImplementation('org.xerial:sqlite-jdbc:3.34.0')
+    testCompile 'org.projectlombok:lombok:1.18.18'
```

Comment:
Lombok should already be set as a dependency via the root [`build.gradle`](https://github.com/MarquezProject/marquez/blob/main/build.gradle#L76).